src/Foundatio.Redis.Signed/Foundatio.Redis.Signed.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\build\common.props" />
+  
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.5;net46</TargetFrameworks>
+    <PackageTags>Queue;Messaging;Message;Bus;MessageBus;Locking;Lock;Distributed;File;Storage;Metrics;Stats;Redis;broker;NETSTANDARD;Core</PackageTags>
+  </PropertyGroup>
+  
+  <ItemGroup Label="Build">
+    <Compile Remove="..\Foundatio.Redis\obj\**\*" />
+    <Compile Remove="artifacts\**" />
+    <Compile Include="..\Foundatio.Redis\**\*.cs;" Exclude="..\Foundatio.Redis\**\AssemblyInfo.cs;..\Foundatio.Redis\obj\**\*;bin\**;obj\**;packages\**" />
+  </ItemGroup>
+  
+  <PropertyGroup Label="Signing">
+    <SignAssembly>true</SignAssembly>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+    <AssemblyOriginatorKeyFile>..\Exceptionless.snk</AssemblyOriginatorKeyFile>
```

Comment:
I'm honestly not sure how the build is passing right now as the snk file needs to be copied to the repo.